modules/microshift-install-rpm-preparing.adoc

```diff
@@ -12,6 +12,8 @@ Before installing {product-title} from an RPM package, you must configure your {
 
 To configure a volume group (VG) that allows LVMS to create the LVs for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime.
 
+The default name of the VG is `rhel`. You can change the VG name during your {op-system} configuration and use it during your installation.
```

Comment:
```suggestion
The default name of the VG is `rhel`. You can change the VG name during your {op-system} configuration and use it during your {product-title} installation.
```

Reply:
I thought maybe the above would make it a bit more clear that you have to do something different during MicroShift installation if you change it. YMMV though :)